bitnami/influxdb/templates/serviceaccount.yaml

```diff
@@ -9,11 +9,11 @@ kind: ServiceAccount
 metadata:
   name: {{ include "influxdb.serviceAccountName" . }}
   namespace: {{ include "common.names.namespace" . | quote }}
-  labels: {{- include "common.labels.standard" ( dict "customLabels" .Values.commonLabels "context" $ ) | nindent 4 }}
+  labels: {{- include "common.labels.standard" (dict "customLabels" .Values.commonLabels "context" .) | nindent 4 }}
```

Comment:
Missing part-of

Reply:
addressed at https://github.com/bitnami/charts/pull/33743/commits/859c509824da5be460e8f02a9701f49e440ae834